### ISSUE TITLE
Enhancement: Add swipe to dismiss

### ISSUE THREAD

**Issue by MobileMon** [boyan01/overlay_support#50]
When showing from the top, it would be nice if the user could swipe up to dismiss

**Comment by boyan01** [comment]
I think it would be a good idea. 

If you have more details to share, I hope you can write it down here.

**Comment by pretorean** [comment]
The notification.dart file has a SlideDismissible object, inside it there is a Dismissible object.
It should be possible to pass the direction property to Dismissible when calling showSimpleNotification.

**Comment by mhstoller** [comment]
I believe that this is already possible, here's sample code of how I am achieving this behavior:
```
 showOverlayNotification(
        (context) => SafeArea(
          child: Dismissible(
              key: UniqueKey(),
              direction: DismissDirection.horizontal,
              onDismissed: (direction) {
                OverlaySupportEntry.of(context).dismiss(animate: false);
              },
              child: Container(
                color: Colors.red,
                padding: EdgeInsets.all(10),
                child: Material(
                  color: Colors.transparent,
                  child: Column(
                    crossAxisAlignment: CrossAxisAlignment.stretch,
                    children: [
                      Text("Test"),
                      Text("Test"),
                      Text("Test"),
                    ],
                  ),
                ),
              )),
        ),
        duration: Duration(seconds: 4),
        position: NotificationPosition.top,
      );
```

Wrap your notification container in a dismissible, and set the onDismissed property to dismiss the overlay with no animation (because dismissible handles the animation already).

**Comment by rish07** [comment]
Hey @boyan01, if no one is working on this issue. I would like to contribute.

**Comment by boyan01** [comment]
> Hey @boyan01, if no one is working on this issue. I would like to contribute.

Pull requests are welcome.  It would be nice if you want to contribute...

**Comment by rish07** [comment]
@boyan01 I checked that showSimpleNotification already has an argument 'slideDismiss' to make notification dismissable. Can you please guide me what this particular issue is referring to?

**Comment by boyan01** [comment]
> Can you please guide me what this particular issue is referring to?

according to the issue detail,  **swipe up or down** to dimiss the notification should be the desired function.

**Comment by boyan01** [comment]
Fixed in 1.2.0. 

introduced a new paramater `slideDismissDirection ` for `showSimpleNotification`: 
https://github.com/boyan01/overlay_support/blob/91aab5281864ace182920d76b3a743bed030c38c/lib/src/notification/overlay_notification.dart#L92-L95
